install/space_checker_utils.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import division
+import re
+import os
+import subprocess
+
+def get_wget_download_size(command):
+    """Get the whole download size.
+
+    :param str command: Wget command.
+
+    :return: The actual download size.
+    :rtype: int
+    """
+    try:
+        _, args = command.split(' ', 1)
+    except IndexError:
+        return 0
+
+    download_size_command = "wget --spider --server-response " + args
+    proc = subprocess.Popen(download_size_command.split(), stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT)
```

Comment:
```python
proc = subprocess.Popen(
    download_size_command.split(),
    stdout=subprocess.PIPE,
    stderr=subprocess.STDOUT)
```

This is actually a personal preference. It is easier to only add one more indentation level when organizing the parameters of a long function call. It is also easier to read than read 2 parameters, then read 1 parameter.

Not mandatory, just a feeling from previous experience with pep8